### PR TITLE
buffs horrorform again

### PR DESCRIPTION
**Pull request by ShadowDeath6**
beefy 
also adam said it sucked so it must be true

:cl: ShadowDeath6
tweak: Increases Horrorform speed very slightly.
tweak: Decreases cooldown for Horrorform Screech.
tweak: Decreases Horrorform chemical drain.
rscadd: Shambling Abominations from the changeling Horrorform now heal passively.
rscdel: Removed Horrorform fleshmend ability.
/:cl:

### CODE REVIEW

**Comment by Cruix** [comment]
Stay tuned for next update when shambling abominations get holoparasites and ethereal jaunt!

**Comment by ShadowDeath6** [comment]
don't give away my schedule

**Comment by Super3222** [comment]
Can you send them "EAT HUMANS TO STAY IN YOUR FORM" when they turn into an abomination? It goes over players heads, and I don't think anyone even knows about that trait.

File: code/modules/mob/living/carbon/alien/humanoid/humanoid.dm [hunk 1]
```diff
@@ -31,6 +31,13 @@
 
 /mob/living/carbon/alien/humanoid/attack_hulk(mob/living/carbon/human/user)
 	if(user.a_intent == "harm")
+		if(M.dna.species.id == "abomination")
```
Comment (ShadowDeath6) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: > M rather than user

Y-you're a fuckin idiot

**Comment by ShadowDeath6** [comment]
MUH

MERGEBEG

@X-TheDark 
@Cruix

File: code/game/gamemodes/changeling/powers/horrorform.dm [hunk 2]
```diff
@@ -80,13 +83,22 @@
...
-				user.mind.spell_list += new /obj/effect/proc_holder/spell/targeted/abomination/abom_fleshmend
+				//user.mind.spell_list += new /obj/effect/proc_holder/spell/targeted/abomination/abom_fleshmend //replaced with constant healing, hopefully not too op
 				user.mind.spell_list += new /obj/effect/proc_holder/spell/targeted/abomination/devour
 				user.mind.spell_list += new /obj/effect/proc_holder/spell/targeted/abomination/abom_revert
 
-//forced reversion
+//forced reversion and healing
 
 /datum/species/abomination/spec_life(mob/living/carbon/human/user)
+	if(user.health < 100 && prob(40))
```
Comment (Super3222) [inline code/game/gamemodes/changeling/powers/horrorform.dm:93]: almost unkillable. almost.

File: code/game/gamemodes/changeling/powers/horrorform.dm [hunk 3]
```diff
@@ -80,13 +83,22 @@
...
+		var/mob/living/carbon/human/H = user
+		H.adjustBruteLoss(-4)
+		H.adjustFireLoss(-4)
+		H.adjustOxyLoss(-10)
+		H.adjustToxLoss(-10)
+		if(prob(25))
```
Comment (Super3222) [inline code/game/gamemodes/changeling/powers/horrorform.dm:99]: what's the point of this check? if you want to show that an abomination is healing, than you should just show it. everytime

Comment (ShadowDeath6) [reply code/game/gamemodes/changeling/powers/horrorform.dm:99]: spam, trust me, it gets annoying when you hit an abomination with an esword and it displays like 5 messages

File: code/game/gamemodes/changeling/changeling.dm [hunk 4]
```diff
@@ -518,7 +518,7 @@ var/list/slot2type = list("head" = /obj/item/clothing/head/changeling, "wear_mas
 	id = "abomination"
 	specflags = list(NOBREATH,COLDRES,NOGUNS,VIRUSIMMUNE,PIERCEIMMUNE,RADIMMUNE,NODISMEMBER)
 	sexes = 0
-	speedmod = 4
+	speedmod = 3
 	armor = 0//has horror armor instead
```
Comment (Super3222) [inline code/game/gamemodes/changeling/changeling.dm:522]: it's default 0.

File: code/modules/mob/living/carbon/alien/humanoid/humanoid.dm [hunk 5]
```diff
@@ -31,6 +31,13 @@
 
 /mob/living/carbon/alien/humanoid/attack_hulk(mob/living/carbon/human/user)
 	if(user.a_intent == "harm")
+		if(user.dna.species.id == "abomination")
+			adjustBruteLoss(40)
```
Comment (Super3222) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm:35]: wait. this makes sense if abominations aren't doing any damage to aliens. but, with this change if they ARE than that's 30 damage + 40 = 70.

Comment (ShadowDeath6) [reply code/modules/mob/living/carbon/alien/humanoid/humanoid.dm:35]: they just regular human punch them currently, that's why I added the change

File: code/game/gamemodes/changeling/changeling.dm (hunk 4 above)
Comment (ShadowDeath6) [reply code/game/gamemodes/changeling/changeling.dm:522]: left it there for the comment so people don't wonder how it has armor

Comment (Super3222) [reply code/game/gamemodes/changeling/changeling.dm:522]: Than make comments and notes above what you're working on. Defining a variable for no reason is bad practice.

File: code/modules/mob/living/carbon/alien/humanoid/humanoid.dm (hunk 5 above)
Comment (Cruix) [reply code/modules/mob/living/carbon/alien/humanoid/humanoid.dm:35]: This is a bit strong but xenos rarely interact with changelings anyway.

**Pull request merged**